src/ucp/rndv/rndv_put.c

```diff
@@ -583,17 +583,23 @@ ucp_proto_rndv_put_mtype_probe(const ucp_proto_init_params_t *init_params)
     size_t frag_size;
     unsigned flags;
     ucp_md_index_t UCS_V_UNUSED dummy_md_id;
+    ucs_memory_type_t frag_mem_type;
     ucp_memory_info_t frag_mem_info;
 
     if (init_params->rkey_config_key == NULL) {
         return;
     }
 
+    frag_mem_type = init_params->rkey_config_key->mem_type;
+    if (!UCS_BIT_GET(context->config.ext.rndv_frag_mem_types, frag_mem_type)) {
+        return;
+    }
```

Comment:
can you pls initialize frag to HOST or first available type from `rndv_frag_mem_types` if HOST is not present there

Reply:
done.